endaq/calc/filters.py

```diff
@@ -4,11 +4,52 @@
 import functools
 
 import pandas as pd
+import numpy as np
 import scipy.signal
 
 from endaq.calc import utils
 
 
+def _get_filter_frequencies_type(low_cutoff, high_cutoff):
+    """Get the filter type and cutoff frequency array."""
+    cutoff_freqs: Union[float, Tuple[float, float]]
+    filter_type: str
+
+    if low_cutoff is not None and high_cutoff is not None:
+        cutoff_freqs = (low_cutoff, high_cutoff)
+        filter_type = "bandpass"
+    elif low_cutoff is not None:
+        cutoff_freqs = low_cutoff
+        filter_type = "highpass"
+    elif high_cutoff is not None:
+        cutoff_freqs = high_cutoff
+        filter_type = "lowpass"        
+    else:
+        return None, None
+        
+    return cutoff_freqs, filter_type        
+
+def rolling_mean(
```

Comment:
One last nit-pick.  Two empty spaces above function definitions, here and a few other places.  Other than that, I think this looks good to go, so consider it approved once all the functions have been formatted.

```suggestion
    return cutoff_freqs, filter_type        


def rolling_mean(
```